src/styles/utils/_colors.scss

```diff
@@ -0,0 +1,7 @@
+$text-color: #060b35;
+$price-color: #616070;
+$buy-botton-color: #ffff;
+$buy-botton-bc-color: #00acdc;
```

Comment:
Similarly, `$buy-botton-bc-color` contains a typo. It should be `$buy-button-bc-color` to ensure correct spelling as per the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).